### DISCUSSION TITLE
Vertical/Portrait CarPlay for "Legacy S/X"

### DISCUSSION THREAD

**Discussion by badxhabit** [tesla-android/issue-tracker#145] [Ideas]
Any chance we can get a vertical display for us "Legacy S/X" folks? :)

I did a mock up and the aspect ratio is an almost identical match.

![IMG_3477](https://user-images.githubusercontent.com/2319140/201391586-f1178058-9a25-4432-b038-48ede1444e87.png)

**Comment by mikegapinski** [comment]
If your car gets updated to a newer version of Tesla software the aspect ratio of the browser will be 1:1 to what we have in the 3/Y and what plaid uses

**Reply by badxhabit** [reply]
How do you mean? My screen will always be vertical, right? I believe I have the latest software update (2022.40.4).

**Reply by badxhabit** [reply]
I am more specifically talking about the vertical CarPlay view that I see in some other versions of cars.

**Reply by mikegapinski** [reply]
I was thinking about this browser layout: 
![Uploading Screenshot 2022-11-10 at 18.29.12.png…]()

I figured that the browser doesn't cover the screen fully on mcu2 legacy s/x

**Reply by badxhabit** [reply]
It doesn't look like your image upload came through.

**Reply by mikegapinski** [reply]
Sorry, that's my punishment for using Safari ;) 
<img width="476" alt="Zrzut ekranu 2022-11-11 o 10 23 33" src="https://user-images.githubusercontent.com/930055/201308968-562b5553-606f-4a21-8c35-c7dc42cc0adc.png">

**Reply by badxhabit** [reply]
ohhhh, ya, I see what you mean.

The only issue is that you can still see the old navigation on top when running Maps or Waze. (See Picture)
![IMG_3479](https://user-images.githubusercontent.com/2319140/201393529-67a491ae-1a58-4ece-bf4a-ee6d50bafa1b.jpeg)




Would a vertical CarPlay give us a full screen?

Also, I just wanted to say thanks again for all your hard work and dedication and that I am super grateful for all the work you have put into this!
